bedrock/firefox/templates/firefox/geolocation.html

```diff
@@ -217,14 +217,6 @@ <h3 data-accordion-role="tab">{{ _('How can I add support for Geolocation to my
 
 </section><!-- end #faq -->
 
-<div class="hidden">
-  <div id="geo-demo">
-    <div id="map_canvas"></div>
```

Comment:
I'm neutering the map on this old page, since it is currently [slated for decommission](https://bugzilla.mozilla.org/show_bug.cgi?id=1430887) anyway. It wouldn't really work with a static map regardless.